bridge-token-factory/src/lib.rs

```diff
@@ -78,6 +81,56 @@ pub enum Role {
     UnrestrictedDeposit,
     UnrestrictedDeployBridgeToken,
     MetadataManager,
+    FeeSetter,
+    FeeClaimer,
+}
+
+#[derive(
+    Default, BorshDeserialize, BorshSerialize, Debug, Clone, Serialize, Deserialize, PartialEq,
+)]
+pub struct DepositTokenBounds {
+    lower_bound: u128,
+    upper_bound: u128,
+}
+
+#[derive(
+    Default, BorshDeserialize, BorshSerialize, Debug, Clone, Serialize, Deserialize, PartialEq,
+)]
+pub struct WithdrawTokenBounds {
+    lower_bound: u128,
+    upper_bound: u128,
+}
+
+#[derive(
+    Default, BorshDeserialize, BorshSerialize, Debug, Clone, Serialize, Deserialize, PartialEq,
+)]
+pub struct DepositFeePercentage {
+    eth_to_near: u128,
+    eth_to_aurora: u128,
+}
+
+#[derive(
+    Default, BorshDeserialize, BorshSerialize, Debug, Clone, Serialize, Deserialize, PartialEq,
+)]
```

Comment:
Please replace `U128` with `u128`, because those types are Serialize/Deserialize into json that doesn't support u128 integers.

Reply:
fixed in 6a7df359a29f6105fae2cccc36ad5277fb747102